app/forms/fields/select_field_with_detail_answer.py

```diff
@@ -28,8 +29,12 @@ def iter_choices(self):
             yield value, label, self.coerce(value) == self.data, detail_answer_id
 
     def pre_validate(self, form):
-        for value, _, _ in self.choices:
-            if value == self.data:
+
+        if not self.choices:
```

Comment:
As mentioned in a different part I don't think this can happen https://github.com/ONSdigital/eq-questionnaire-validator/blob/2e3948aacf6f3c892919649481f2ca45e0036b18/app/validators/answers/option_answer_validator.py#L21